src/main/java/uk/gov/hmcts/reform/idam/client/IdamApi.java

```diff
@@ -62,4 +63,11 @@ AuthenticateUserResponse authenticateUser(
     TokenExchangeResponse exchangeCode(
         @RequestBody ExchangeCodeRequest exchangeCodeRequest
     );
+
+    @GetMapping(
```

Comment:
can't you do?
```suggestion
    @GetMapping("/o/userinfo)
```

Reply:
I think it could be simplified like that (with "/o/userinfo") of course!

Reply:
Yeah, I just followed what was already there but I am happy to change it.